Unity-Technologies/ScriptableRenderPipeline
# 2d/lwrp fix

### Checklist for PR maker
- [X] Have you added a backport label (if needed)? For example, the `need-backport-2019.3` label. After you backport the PR, the label changes to `backported-2019.3`.

---
### Purpose of this PR
Fixes a typo in a 2d lightweight shader include file

---
### Testing status

**Manual Tests**: What did you do?
- Other: Checked to make sure the new include reference would not cause errors when used.


**Yamato**: (Select your branch):

---
### Comments to reviewers
Notes for the reviewers you have assigned.
